clients/python/README.md

```diff
@@ -146,18 +146,19 @@ for version in registry.get_model_versions("my-model"):
     ...
 ```
 
-To customize sorting order or query limits you can also use
+You can also use `order_by_creation_time`, `order_by_update_time`, or `order_by_id` to change the sorting order
 
 ```py
-latest_updates = registry.get_model_versions("my-model").order_by_update_time().descending().limit(20)
+latest_updates = registry.get_model_versions("my-model").order_by_update_time().descending()
 for version in latest_updates:
     ...
 ```
 
-You can use `order_by_creation_time`, `order_by_update_time`, or `order_by_id` to change the sorting order.
+By default, all queries well be `ascending`, but this method is also available for explicitness.
 
-> Note that the `limit()` method only limits the query size, not the actual loop boundaries -- even if your limit is 1
-> you will still get all the models, with one query each.
+> Note: You can also set the `page_size()` that you want to use, but the pager will automatically manage pages for you.
```

Comment:
```suggestion
> Note: You can also explicitly set the `page_size()` that you want this client to use when invoking the remote backend; in any case the pager will automatically manage pages for you.
```

wdyt?

Reply:
it's not "in any case". It truly depends on the user using it as an iterator. If they use the internal methods it's up to them to make it not loop.

Reply:
I also wouldn't say "this client" because each iterator has its own options. So:

```
rm_iter = registry.get_registered_models()
rm_iter.page_size(1)
rm_iter2 = registry.get_registered_models() // new obj with default page size
```

Reply:
s/this client/the iterator/ or s/this client/the pager/ ?